src/app/page.tsx

```diff
@@ -1,12 +1,28 @@
-import BottomNav from '@/components/bottom-nav/BottomNav';
+'use client';
 import styles from './page.module.css';
+import ActiveButton from '@/components/button/ActiveButton';
+import InactiveButton from '@/components/button/InactiveButton';
 
 export default function Home() {
   return (
-    <main className={styles.main}>
-      <h1>Welcome to Deer Tracker!</h1>
-      <h3>Powered by AI & Machine Learning</h3>
-      <BottomNav />
+    <main className={styles.mainpageContainer}>
+      <div className={styles.headerconatiner}>
+        <h1 className={styles.welcomecontainer}>Welcome to Deer Tracker!</h1>
+      </div>
+
+      <div className={styles.lgnbttncontainer}>
+        <ActiveButton
+          text="Log in"
+          onClick={() => console.log('ln Button clicked!')}
+        />
+      </div>
+      <div className={styles.signbttncontainer}>
+        <InactiveButton
+          text="Sign up"
```

Comment:
do the same for this string as with log in, when the time comes to work on create account and login u can just add those variables directly in those pages too. 

The usefulness here is that if you wanted to change the text from say "sign up" to "create account", you can do that just within constants and it changes everywhere, instead of changing multiple files for one string.